### PR TITLE
 Fix S3 and server_certificate integration specs  

### PR DESCRIPTION
**Pull request by harikesh-kolekar**
Signed-off-by: Harikesh Kolekar <harikesh.kolekar@msystechnologies.com>

### CODE REVIEW

File: lib/chef/resource/aws_server_certificate.rb [hunk 1]
```diff
@@ -11,9 +11,9 @@ class Chef::Resource::AwsServerCertificate < Chef::Provisioning::AWSDriver::AWSR
 
   def aws_object
     begin
-      cert = driver.iam.server_certificates[name]
-      # this will trigger a ::Aws::IAM::Errors::NoSuchEntity if the cert does not exist
-      cert.arn
+      cert = ::Aws::IAM::ServerCertificate.new(name,{client: driver.iam} )
+      #cert = driver.iam.server_certificates[name]
```
Comment (NimishaS) [inline lib/chef/resource/aws_server_certificate.rb]: Please remove the commented out line.

File: spec/integration/aws_server_certificate_spec.rb [hunk 2]
```diff
@@ -4,97 +4,79 @@
...
+P/KvXRF9AtEQe27/CWnJywCkP6tT6baZM6X+GGgAPUHvxN4BmJzz6uHpMVH+rBbb
+t9ruLoSdX0zbaTRLesBC5Hc8uK2wzvDx0pUj+aKcWg5mtPBT6yReH6D5ePV2Jf10
+9FGKMqPN6tOO6ZyAIWuKx3v09JzxmWGxNEyR65SNiI+ft092UFEKXYfgK58HZlWj
+pBcOsHECAwEAATANBgkqhkiG9w0BAQ0FAAOCAQEAY1KXZv35hUER0WZz7JMKlvhI
+IUpfB2NibP9G5LhtxFY1fa+MLp9mJ+yI3hg0x+6xBPTiPDGVpFdJ+LH/mKWxsaWT
+17ZRGOeG2gZAlGr8Y8IMuR5cv2fmG6zCObMFxFR89pKvAYR+H5RFcHBg/N86SX6D
+/KKNXu+NC9z/lrNVU6J/XUCk54YdmjMvadHs1aJp6NWH7LI9df27AlBGLnRc/w04
+agCh4aCsjczD2YeBVl6idws/InYbSwhz7x9zXz2qB3BbI3psgBfJQQcJukulVGza
+RkD993U+CJpJMDbpIGRe9lXe33R3tGbbvfa4FaXZlZgoKCrE21SCb2hxlNG2+w==
+-----END CERTIFICATE-----
```
Comment (NAshwini) [inline spec/integration/aws_server_certificate_spec.rb]: extra spaces are been added after -----END CERTIFICATE-----

File: spec/integration/aws_server_certificate_spec.rb [hunk 3]
```diff
@@ -4,97 +4,79 @@
...
+MDruVLTXOg8XuN1RW4c8SmauaE2KXaYNUuzZXv0CgYEAwTy38tyfrIcDWxUut2ep
+skrGABZCLVeK3Sc+IHFZfM1aQnufccbF+2h5KzLCrmDj48HdvnbzS/maNTzq45J4
+QM2PaJjtObmo3QUIOEZ+2oZcSYjY/dO2sTY5uh4ghLEOyboGlYWGkLG57JnKU60j
+9NZqtqZyfsUaOWQ2TeOdP8ECgYEAxW6J4T32096xag6L7pC6SmZIMg3m0LpxaxaX
+k7JouTKFS7IMwTW3AFpyHz+KG4QcBoQj94ofZvapIOv8E5+8MkSVyuONRbHuoOeE
+/RkCYbmbwUxJ2m2w4uL62VWCPxqURm2VvnQHXNM+Etkaf3O5v96PcmQVbFBovhzt
+DNbJssECgYBIAFB3J66cZH3zBfyanHwYMPlTupxUBTsN19UCAK7MZqA5kbqG7b7Q
+cKPrgR7gibLvaxj22lvD8uEICMGy6g8s3+PyDwcw5PNrKOdIkRrYQxTFtfs1JRCa
+6kM8q32f1I3qd31zbpoDBHq+Se2RSaLbmoIAFqDSCWyeUHydALo2kw==
+-----END RSA PRIVATE KEY-----
```
Comment (NAshwini) [inline spec/integration/aws_server_certificate_spec.rb]: same as above as extra spaces..

Comment (NAshwini) [inline spec/integration/aws_server_certificate_spec.rb:59]: same as above as extra spaces..

File: spec/integration/aws_server_certificate_spec.rb (hunk 2 above)
Comment (NAshwini) [inline spec/integration/aws_server_certificate_spec.rb:29]: extra spaces are been added after -----END CERTIFICATE-----

**Review by dheerajd-msys — APPROVED** [review]

File: spec/integration/aws_server_certificate_spec.rb (hunk 2 above)
Comment (harikesh-kolekar) [reply spec/integration/aws_server_certificate_spec.rb:29]: It's part of server certificate so can't remove it

File: spec/integration/aws_server_certificate_spec.rb (hunk 3 above)
Comment (harikesh-kolekar) [reply spec/integration/aws_server_certificate_spec.rb:59]: It's part of server certificate so can't remove it

**Review by NimishaS — APPROVED** [review]

**Pull request merged**